### PR TITLE
Update German for informal style

### PR DESCRIPTION
**Pull request by Eitot**
Apple switched to an [informal style in macOS Sierra](http://www.mactechnews.de/news/article/macOS-Sierra-redet-den-Nutzer-mit-Du-an-164371.html
). There were two untranslated strings as well.

### CODE REVIEW

File: Localizations/KeepingYouAwake/de.xliff [hunk 1]
```diff
@@ -154,10 +154,12 @@ Rechtsklick, um das Menü anzuzeigen</target>
       </trans-unit>
       <trans-unit id="Disable menu bar icon highlight color">
         <source>Disable menu bar icon highlight color</source>
+        <target>Auswahlfarbe im Statusmenü nicht anzeigen</target>
```
Comment (newmarcel) [inline Localizations/KeepingYouAwake/de.xliff]: I would reword this a bit to e.g.

> Keine Auswahlfarbe in der Menüleiste anzeigen

Comment (Eitot) [inline Localizations/KeepingYouAwake/de.xliff]: I changed it.

Comment (Eitot) [inline Localizations/KeepingYouAwake/de.xliff]: The formulation is still not very specific. It probably should be something like ‘Keine Auswahlfarbe beim Klicken des Statusmenüs anzeigen’.

**Comment by newmarcel** [comment]
Thanks 👍

I didn't even notice that macOS uses _"du"_ now everywhere 😉
